utils/main.go

```diff
@@ -88,6 +89,20 @@ func CheckExpired(expireOn string) (bool, error) {
 
 }
 
+// ValidateUserKey checks for the presence of a characters which should not be
+// allowed in a username string, returns an error if any such characters are
+// detected, nil otherwise.
+// Currently, the only banned characters are whitespace characters.
+func ValidateUserKey(user string) error {
+	whiteSpaceRegexp, _ := regexp.Compile("\\s")
+
+	hasWhiteSpace := whiteSpaceRegexp.MatchString(user)
+	if hasWhiteSpace {
```

Comment:
these lines can be combined

Reply:
Changed: now reads with hasWhiteSpace == true for readability